.azure-pipelines/ci-build.yml

```diff
@@ -148,14 +148,6 @@ extends:
               projects: src/Microsoft.OpenApi.OData.Reader/Microsoft.OpenAPI.OData.Reader.csproj
               arguments: '-o $(Build.ArtifactStagingDirectory) --configuration $(BuildConfiguration) --no-build --include-symbols --include-source /p:SymbolPackageFormat=snupkg'
 
-          - task: PowerShell@2
```

Comment:
can you provide more context on why we're removing that validation step?

Reply:
I think in this case, it makes sense to drop the check and align with the openApi.Net library.
https://github.com/microsoft/OpenAPI.NET/blob/main/.azure-pipelines/ci-build.yml

Furthermore, since we now use release please and the deploy pipeline would run off of a tag and not every check-in on main, we have the confidence that the tagged run is an intended release (while previously, we would use this script as a filter for releasing). 

What do you think?

Reply:
sure, I'm not pushing back, simply outlining the fact that providing this rationale in the PR description would have been great!
Also we should create a reflection of that PR for support/v1